test/chunking-form/samples/external-independently-renamed/_config.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+let i = 0;
+
+module.exports = {
+	description: 'imports of the same external can be independently renamed',
+	options: {
+		input: ['main1.js', 'lib/main2.js'],
+		external(id) {
+			return id.endsWith('placeholder');
+		},
+		paths() {
```

Comment:
Currently this only receives `id`, so I use an int to fake knowing the difference between the relative files. Once #2277 lands, this will have `parent` as well as `id`, so you can determine the correct relative import.